backend/app/api/routes/auth.py

```
@@ -1,226 +1,448 @@
+            "enabled": True
+        })
+
+    return {"providers": providers}
+
+
+# Password management endpoints
+@router.post("/change-password")
+async def change_password(
+    current_password: str,
+    new_password: str,
+    current_user = Depends(get_current_active_user_dependency)
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)